subsys/bluetooth/host_extensions/radio_notification_conn_cb.c

```diff
@@ -151,30 +132,48 @@ int bt_radio_notification_conn_cb_register(const struct bt_radio_notification_co
 	return 0;
 }
 
-static int driver_init(const struct device *dev)
+#if defined(CONFIG_SOC_COMPATIBLE_NRF52X) || defined(CONFIG_SOC_COMPATIBLE_NRF5340_CPUNET)
+static int set_sys_clock_start_to_bt_clk_start_us(void)
```

Comment:
would it make sense to expose this function? Then it could be used in the time sync samples, as well.

Reply:
I agree. This would fit in a header named `mpsl_time.h` or similar